tests/outlineCompiler_test.py

```diff
@@ -1046,7 +1046,11 @@ def test_custom_layer_compilation_interpolatable(layertestrgufo, layertestbdufo)
         "edotabove",
     ]
 
-    sparse_tables = [tag for tag in master_ttfs[1].keys() if tag != "GlyphOrder"]
+    sparse_tables = [
+        tag
+        for tag in master_ttfs[1].keys()
+        if tag not in ("GlyphOrder", "cvt ", "fpgm", "prep")
```

Comment:
should "cvt ", "fpgm", "prep" be added to SPARSE_TTF_MASTER_TABLES list?

Reply:
That makes sense.